src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/02 Client.md

```diff
@@ -0,0 +1,95 @@
+};
+
+```
+
+</CodeGroup>
+
+
+## HTTP module functions equivalents
+
+The `Client` class exposes the same API as the standard `http` module. Except for the `batch` method, which is absent from `Client`. The following table lists the `Client` methods which have an equivalent in the standard `http` module:
+
+| Method | HTTP equivalent | Description |
+| :---------------------------------------------- | :------------------------------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Client.del(url, [body], [params])` | [http.del](/javascript-api/k6-http/del) | Performs an instrumented HTTP DELETE request. The method has the same prototype as the `http.del` function and should transparently act as a drop-in replacement for it. |
+| `Client.get(url, [params])` | [http.get](/javascript-api/k6-http/get) | Performs an instrumented HTTP GET request. The method has the same prototype as the `http.get` function and should transparently act as a drop-in replacement for it. |
+| `Client.head(url, [params])` | [http.head](/javascript-api/k6-http/head) | Performs an instrumented HTTP HEAD request. The method has the same prototype as the `http.head` function and should transparently act as a drop-in replacement for it. |
+| `Client.options(url, [body], [params])` | [http.options](/javascript-api/k6-http/options) | Performs an instrumented HTTP OPTIONS request. The method has the same prototype as the `http.options` function and should transparently act as a drop-in replacement for it. |
+| `Client.patch(url, [body], [params])` | [http.patch](/javascript-api/k6-http/patch) | Performs an instrumented HTTP PATCH request. The method has the same prototype as the `http.patch` function and should transparently act as a drop-in replacement for it. |
+| `Client.post(url, [body], [params])` | [http.post](/javascript-api/k6-http/post) | Performs an instrumented HTTP POST request. The method has the same prototype as the `http.post` function and should transparently act as a drop-in replacement for it. |
+| `Client.put(url, [body], [params])` | [http.put](/javascript-api/k6-http/head) | Performs an instrumented HTTP HEAD request. The method has the same prototype as the `http.put` function and should transparently act as a drop-in replacement for it. |
+| `Client.request(method, url, [body], [params])` | [http.request](/javascript-api/k6-http/request) | Performs an instrumented HTTP request. The method has the same prototype as the `http.request` function and should transparently act as a drop-in replacement for it. |
+| `Client.asyncRequest(method, url, [body], [params])` | [http.asyncRequest](/javascript-api/k6-http/asyncRequest) | Performs an instrumented HTTP asynchronous request. The method has the same prototype as the `http.asyncRequest` function and should transparently act as a drop-in replacement for it. |
```

Comment:
```suggestion
| `Client.asyncRequest(method, url, [body], [params])` | [http.asyncRequest](/javascript-api/k6-http/asyncrequest) | Performs an instrumented HTTP asynchronous request. The method has the same prototype as the `http.asyncRequest` function and should transparently act as a drop-in replacement for it. |
```

Paths are always lower case ... it seems

Reply:
My bad 🤦🏻 cheers!